sdk/anomalydetector/ai-anomaly-detector-rest/src/clientDefinitions.ts

```diff
@@ -2,146 +2,194 @@
 // Licensed under the MIT license.
 
 import {
-  DetectUnivariateEntireSeriesParameters,
-  DetectUnivariateLastPointParameters,
-  DetectUnivariateChangePointParameters,
-  GetMultivariateBatchDetectionResultParameters,
-  CreateAndTrainMultivariateModelParameters,
-  ListMultivariateModelsParameters,
-  DeleteMultivariateModelParameters,
-  GetMultivariateModelParameters,
-  DetectMultivariateBatchAnomalyParameters,
-  DetectMultivariateLastAnomalyParameters,
+  UnivariateDetectUnivariateEntireSeriesParameters,
```

Comment:
Currently we levereage the namespace as the prefix of paramter name. But the swagger one is not. 

@qiaozha Do you think we need to fix it?

Reply:
Not sure if it will  cause other existing scenarios to fail if we fix it.  @joheredi what do you think ?